h2o-r/h2o-package/R/modelselection.R

```diff
@@ -113,10 +113,10 @@
 #' prostate_path <- system.file("extdata", "prostate.csv", package = "h2o")
 #' prostate <- h2o.uploadFile(path = prostate_path)
 #' prostate$CAPSULE <- as.factor(prostate$CAPSULE)
-#' model <- h2o.modelselection(y = "VOL", x = c("CAPSULE", "RACE", "AGE", "RACE", "DPROS", "DCAPS", "PSA", "GLEASON:), training_frame = prostate)
+#' model <- h2o.modelselection(y = "VOL", x = c("CAPSULE", "RACE", "AGE", "RACE", "DPROS", "DCAPS", "PSA", "GLEASON"), training_frame = prostate)
```

Comment:
This happens automatically

Reply:
@wendycwong , this should not happen automatically. If you change the `modelselection` to `modelSelection`, the example would not work now.

Reply:
@maurever I made the fix for changing modelselection to modelSelection in gen_modelselection.py for custom/R.  Then, I did a ./gradlew build -x test.  This will then generate ModelSelection.R and the example is fixed.  I will need to add both gen_modelselection.py and the newly generated ModelSelection.R. 

Since the ModelSelection.R file is automatically generated, that is the automatic part I am referring to.  if you look at the latest code, it has been fixed.  Thanks dear.